Client/Frontend/Menu/HistoryViewController.swift

```diff
@@ -117,63 +117,12 @@ class HistoryViewController: SiteTableViewController {
     
     cell.imageView?.contentMode = .scaleAspectFit
     cell.imageView?.image = FaviconFetcher.defaultFavicon
+    cell.imageView?.layer.borderColor = UIColor(white: 0, alpha: 0.2).cgColor
```

Comment:
I would extract `UIColor(white: 0, alpha: 0.2)` somewhere, we use it in few places